src/vm/jitinterface.cpp

```diff
@@ -10255,7 +10255,7 @@ void InlinedCallFrame::GetEEInfo(CORINFO_EE_INFO::InlinedCallFrameInfo *pInfo)
 {
     LIMITED_METHOD_CONTRACT;
 
-    pInfo->size                          = sizeof(GSCookie) + sizeof(InlinedCallFrame);
+    pInfo->size                          = READYTORUN_RUNTIME_CONSTANT_PInvokeTransitionFrameSize;
```

Comment:
This should stay as is. Instead, the constant should be used in CEEInfo::getEEInfo instead of `11` that is hardcoded there today.